lib/remote/actionshandler.cpp

```diff
@@ -56,9 +56,14 @@ bool ActionsHandler::HandleRequest(
 
 		try {
 			objs = FilterUtility::GetFilterTargets(qd, params, user);
-		} catch (const std::exception& ex) {
```

Comment:
`std::exception` should still be caught somewhere.

Reply:
But isn't "catch all" an anti pattern? Imo the only valid cases are if you call into unknown, untrusted 3rd party code (Boost is not in that class), or at the outermost layer of your application (enclosing the main loop).

Reply:
I haven't checked too much context of this code right now, so maybe there are already some more safeguards somewhere else that achieve the following. But I think the goal should be that if an HTTP request causes any (unexpected) exception, that should properly return a 500 Internal Server Error response and not tear down the full process.